packages/react/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,198 @@
+  },
+  [`&.${tagClasses.colorWarning}`]: {
+    '--background': theme.vars.palette.warning[300],
+    '--text': theme.vars.palette.monoB[500],
+    '--hovered': theme.vars.palette.monoB.A50,
+    '--pressed': theme.vars.palette.monoB.A150,
+  },
+  [`&.${tagClasses.colorError}`]: {
+    '--background': theme.vars.palette.error[300],
+    '--text': theme.vars.palette.monoB[500],
+    '--hovered': theme.vars.palette.monoB.A50,
+    '--pressed': theme.vars.palette.monoB.A150,
+  },
+  [`&.${tagClasses.colorSuccess}`]: {
+    '--background': theme.vars.palette.success[300],
+    '--text': theme.vars.palette.monoB[500],
+    '--hovered': theme.vars.palette.monoB.A50,
+    '--pressed': theme.vars.palette.monoB.A150,
+  },
+  [`&.${tagClasses.colorMonoA}`]: {
+    '--background': theme.vars.palette.monoA[500],
+    '--text': theme.vars.palette.monoB[500],
+    '--hovered': theme.vars.palette.monoB.A50,
+    '--pressed': theme.vars.palette.monoB.A150,
+  },
+  [`&.${tagClasses.colorMonoB}`]: {
+    '--background': theme.vars.palette.monoB[500],
+    '--text': theme.vars.palette.monoA[500],
+    '--hovered': theme.vars.palette.monoA.A50,
+    '--pressed': theme.vars.palette.monoA.A150,
+  },
+  [`&.${tagClasses.colorWhite}`]: {
+    '--background': theme.vars.palette.white[500],
+    '--text': theme.vars.palette.black[500],
+    '--hovered': theme.vars.palette.black.A50,
+    '--pressed': theme.vars.palette.black.A150,
+  },
+  [`&.${tagClasses.colorBlack}`]: {
+    '--background': theme.vars.palette.black[500],
+    '--text': theme.vars.palette.white[500],
+    '--hovered': theme.vars.palette.white.A50,
+    '--pressed': theme.vars.palette.white.A150,
+  },
+})) as typeof ButtonBase;
+
+const TagStartIcon = styled('span', {
+  name: 'ESTag',
+  slot: 'StartIcon',
+  overridesResolver: (_props, styles) => [styles.icon, styles.startIcon],
+})(() => ({
+  display: 'inherit',
+  color: 'var(--icon)',
+}));
+
+const TagEndIcon = styled('span', {
+  name: 'ESTag',
+  slot: 'EndIcon',
+  overridesResolver: (_props, styles) => [styles.icon, styles.endIcon],
+})(() => ({
+  display: 'inherit',
+  color: 'var(--icon)',
+}));
+
+/**
+ * The Tag component is used to categorize content.
+ */
+export const Tag = (inProps: TagProps) => {
+  const {
+    children,
+    classes: inClasses,
+    className,
+    sx,
+    clickable = false,
+    color = 'primary',
+    startIcon,
+    endIcon,
+    variant = 'overline',
+    ...props
+  } = useThemeProps({
+    props: inProps,
+    name: 'ESTag',
+  });
+
+  const ownerState = { inClasses, variant, color, clickable, ...props };
+  const classes = useUtilityClasses(ownerState);
+
+  const component = clickable ? ButtonBase : 'div';
+
+  return (
+    <TagRoot className={clsx(classes.root, className)} component={component} sx={sx} {...props}>
+      {!!startIcon && <TagStartIcon className={classes.startIcon}>{startIcon}</TagStartIcon>}
+      <Typography variant={variant}>{children}</Typography>
```

Comment:
Тут бы добавить, чтобы всегда одинаковый элемент был
```suggestion
      <Typography component="span" variant={variant}>{children}</Typography>
```